docker/e2e.command.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
 
-yarn install
 yarn db:migration:run
-yarn test:e2e
+yarn test:e2e --force-exit
```

Comment:
I guess this is the only workaround for now until we find the real issue for `jest` not exiting correctly.